dags/send_to_teams_dag.py

```diff
@@ -0,0 +1,43 @@
+import pendulum
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+
+from image_loader import load_image
+from quote_loader import load_quote
+from teams_notifier import send_to_teams
+
+with DAG(
+        dag_id="agile_project_MSI_2",
+        description="A simple DAG to send daily quote with image to Teams",
+        start_date=pendulum.today(),
+        schedule_interval="@daily",
```

Comment:
Do you think that this is a good idea to send somebody messages at the middle of a night?)